Boolean.CSharp.Test/CoreTests.cs

```diff
@@ -1,24 +1,125 @@
-﻿using Boolean.CSharp.Main;
+﻿using System.Security.Principal;
+using Boolean.CSharp.Main;
 using NUnit.Framework;
 
 namespace Boolean.CSharp.Test
 {
-    [TestFixture]
     public class CoreTests
     {
-        private Core _core;
 
-        public CoreTests()
+
+
+        [Test]
+        public void addCustomer()
+        {
+            Bank bank = new Bank();
+            Customer bob = new Customer("Bob");
+            bool customerTest = bank.addCustomer(bob);
+
+            Assert.IsTrue(customerTest);
```

Comment:
you are not testing whether the Bank has actually stored the Customer in the customer list...